### DISCUSSION TITLE
BTC Signing Error

### DISCUSSION THREAD

**Discussion by akhilccrb** [trustwallet/wallet-core#2163] [Q&A]
![Screenshot 2022-04-22 at 11 52 41 AM](https://user-images.githubusercontent.com/45847512/164615600-7d9b3e3f-9abf-49cb-b97c-6aac5728da10.png)

Can you please check ?

**Comment by akhilccrb** [comment]
BitcoinScript has no member **buildForAddress**

**Comment by hewigovens** [comment]
Is this sample code from https://developer.trustwallet.com/wallet-core/integration-guide/wallet-core-usage? will update it soon

**Comment by akhilccrb** [comment]
So we can't Signin BTC Now ? Is there any alternatives ?

**Comment by hewigovens** [comment]
Please replace `buildForAddress` with `lockScriptForAddress`, it's just the document issue

**Comment by akhilccrb** [comment]
![Screenshot 2022-04-22 at 12 44 33 PM](https://user-images.githubusercontent.com/45847512/164625386-48cb1305-2347-481f-b31a-4f092da3adc5.png)

**Comment by akhilccrb** [comment]
**Error validating transaction: Error running script for input 0 referencing f30944c3f6a0670f4e732a98079d2ddfa22b312fbb4c96fad225b21cfbf5c235 at 1: Script was NOT verified successfully..**


This is the error after using  _**lockScriptForAddress**_

**Comment by hewigovens** [comment]
can you share your full sample code without private key? hard to tell what's wrong
